docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+
+```python
+SESSION_COOKIE_SAMESITE = "Lax"
+```
+
+### APP_ALLOWED_HOSTS
+
+List of allowed hosts for the application.
+
+```python
+APP_ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+```
+
+### SPA_HOST
+
+Host URL for Single Page Application.
+
+```python
+SPA_HOST = "https://127.0.0.1:3000"
+```
+
+### SPA_PATHS
+
+Paths and placeholders for Single Page Application routes.
+
+```python
+SPA_PATHS = dict(
+    literature="/literature/%(pid)s",
+    loan="/backoffice/loans",
+    patron="/backoffice/patrons",
+    profile="/profile",
+    search="/search?q=%(querystring)s",
+)
+```
+
+Here's the continuation and completion of the configuration file documentation in Markdown format:
+
+## Records REST Configuration
+
+### RECORDS_REST_MAX_RESULT_WINDOW
+
+Maximum result window for Records REST endpoints.
+
+```python
+RECORDS_REST_MAX_RESULT_WINDOW = 10000
+```
+
+### RECORDS_REST_VOCAB_MAX_RESULT_WINDOW
+
+Maximum result window for vocabulary Records REST endpoints.
+
+```python
+RECORDS_REST_VOCAB_MAX_RESULT_WINDOW = 500
+```
+
+### PIDSTORE_RECID_FIELD
+
+Field name in the PID store for the record identifier.
+
+```python
+PIDSTORE_RECID_FIELD = "pid"
+```
+
+### REST_MIMETYPE_QUERY_ARG_NAME
+
+Name of the URL argument to choose response serializer.
+
+```python
+REST_MIMETYPE_QUERY_ARG_NAME = "format"
+```
+
+### RECORDS_REST_ENDPOINTS
+
+Dictionary defining the endpoints for Records REST API.
+
+```python
+RECORDS_REST_ENDPOINTS = {
+    "docid": {
+        "pid_type": DOCUMENT_PID_TYPE,
+        "pid_minter": DOCUMENT_PID_MINTER,
+        "pid_fetcher": DOCUMENT_PID_FETCHER,
+        "search_class": DocumentSearch,
+        "record_class": Document,
+        "indexer_class": DocumentIndexer,
+        "search_factory_imp": "invenio_app_ils.search_permissions:ils_search_factory",
+        "record_loaders": {
+            "application/json": "invenio_app_ils.documents.loaders:document_loader"
+        },
+        "record_serializers": {
+            "application/json": "invenio_app_ils.literature.serializers:json_v1_response"
+        },
+        "search_serializers": {
+            "application/json": "invenio_app_ils.literature.serializers:json_v1_search",
+            "text/csv": "invenio_app_ils.literature.serializers:csv_v1_search",
+        },
+        "search_serializers_aliases": {
+            "csv": "text/csv",
+            "json": "application/json",
+        },
+        "list_route": "/documents/",
+        "item_route": "/documents/<{0}:pid_value>".format(doc_pid_converter()),
+        "default_media_type": "application/json",
+        "max_result_window": RECORDS_REST_MAX_RESULT_WINDOW,
+        "error_handlers": {},
+        "read_permission_factory_imp": record_read_permission_factory,
+        "list_permission_factory_imp": allow_all,
+        "create_permission_factory_imp": backoffice_permission,
+        "update_permission_factory_imp": backoffice_permission,
+        "delete_permission_factory_imp": backoffice_permission,
```

Comment:
I would leave this and explain that they are used to implement permission policies for specific endpoints